engine/src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -142,29 +152,67 @@ object CromwellRuntimeAttributes {
       case scala.util.Success(x) => x.to(MemoryUnit.GB).amount.successNel
       case scala.util.Failure(x) => x.getMessage.failureNel
     }
+    val walltime = validateWalltime(attributeMap.get(WALLTIME))
+    val queue = validateQueue(attributeMap.get(QUEUE))
     val bootDiskSize = validateBootDisk(attributeMap.get(BOOT_DISK))
-
-    (docker |@| zones |@| failOnStderr |@| continueOnReturnCode |@| cpu |@| preemptible |@| disks |@| memory |@| bootDiskSize) {
-      new CromwellRuntimeAttributes(attributes, _, _, _, _, _, _, _, _, _)
-    } match {
+    
+    /*
+     * scalaz.syntax.ApplicativeBuilder |@| (as of version 7.3, anyhow) runs out of steam at 12
+     * arguments so we can't just keep extending what Broad had here to add queue, walltime, etc.
+     * The chained <*> below basically does the same thing.
+     */
+    val curriedCtor = (CromwellRuntimeAttributes(_: Map[String, WdlValue], _: Option[String], _: Seq[String], _: Boolean, _: ContinueOnReturnCode, _: Int, _: Int, _: Seq[JesAttachedDisk], _: Double, _: String, _: Option[String], _: Int)).curried
+    bootDiskSize <*> (queue <*> ( walltime <*> (memory <*> ( disks <*> ( preemptible <*> ( cpu <*> (continueOnReturnCode <*> ( failOnStderr <*> ( zones <*> (docker map curriedCtor(attributes))))))))))) match {
```

Comment:
Not at all required for this PR, but if you enjoy looking at mind-bending type-level code for boilerplate removal, some of this might be useful here someday:

http://meta.plasm.us/posts/2013/06/05/applicative-validation-syntax/